companion/src/helpers.cpp

```diff
@@ -834,7 +832,8 @@ void startSimulation(QWidget * parent, RadioData & radioData, int modelIdx)
 
     if (board == BOARD_HORUS && HORUS_READY_FOR_RELEASE()) {
       dialog = new SimulatorDialogHorus(parent, simulator, flags);
-      GetEepromInterface()->saveFile(*simuData, g.profile[g.id()].sdPath());
+      SdcardFormat sdcard(g.profile[g.id()].sdPath());
```

Comment:
Here is the place where we will use the alternate (temporary dir) path to fully implement #4231